src/components/formGroup/examples/demo/Radios.vue

```diff
@@ -21,6 +21,40 @@
       >Sony</cdr-radio>
 
     </cdr-form-group>
+    <cdr-text>Selected Radio: {{ex}}</cdr-text>
+
+    <cdr-form-group 
+      label="What's your favorite required radio?" 
+      id="favorite-required-radio"
+      :error="hasError"
+      aria-describedby="errorStatus-radio"    
+    >
+
+      <cdr-radio
+        name="required-radio"
+        custom-value="casio"
+        v-model="ex2"
+        @change="validate"
+      >Casio</cdr-radio>
+
+      <cdr-radio
+        name="required-radio"
+        custom-value="panasonic"
+        v-model="ex2"
+       @change="validate"
+      >Panasonic</cdr-radio>
+
+      <cdr-radio
+        name="required-radio"
+        custom-value="sony"
+        v-model="ex2"
+       @change="validate"
+      >Sony</cdr-radio>
+      <template #error>
+        <span id="errorStatus-radio">You must make a selection!</span>
+      </template>
+    </cdr-form-group>
+    <cdr-text>Selected Radio: {{ex2}}</cdr-text>
 
     <cdr-form-group
       label="Disabled Radio Example"
```

Comment:
I find our disabled example odd - if the entire form group is disabled why are we seeing the hover effect on the checkboxes / radios